### PR TITLE
OM-14 Include support for regressors

### PR DESCRIPTION
**Pull request by carloszanella**
WIP

### CODE REVIEW

File: omigami/models/sklearn_estimator.py [hunk 1]
```diff
@@ -73,11 +81,15 @@ def feature_importances(self):
...
 
     def set_random_state(self, random_state: RandomState):
         for _, step in self._estimator.steps:
             try:
                 step.set_params(random_state=random_state)
             except ValueError:
                 pass  # not all the elements of the pipeline have to be random
+
+    def __repr__(self):
+        return f"SKLearnPipeline(n_steps={self._estimator.steps})"
```
Comment (kayibal) [inline omigami/models/sklearn_estimator.py:95]: n_step == steps?

File: omigami/models/sklearn_estimator.py [hunk 2]
```diff
@@ -58,6 +63,9 @@ def fit(self, X, y):
     def predict(self, X):
         return self._estimator.predict(X)
 
+    def __repr__(self):
+        return f"SKLearnEstimator(model={self._estimator.__class__.__name__})"
```
Comment (kayibal) [inline omigami/models/sklearn_estimator.py:67]: How about this?
```suggestion
        return f"SKLearnEstimator(model={self._estimator.__repr__()})"
```

**Review by kayibal — COMMENTED** [review]
So far looks good to me surprising how easy it was. One thing: I guess if scoring function isn't fit for regression we will just wait for it to raise right when it is evaluated for the first time?

File: omigami/models/sklearn_estimator.py (hunk 1 above)
Comment (carloszanella) [reply omigami/models/sklearn_estimator.py:95]: :+1:

File: tests/test_estimator.py [hunk 3]
```diff
@@ -13,8 +12,10 @@
     "RFC",
     "XGBC",
     "PLSC",
-    SVC(random_state=1),
-    Pipeline([("norm", Normalizer()), ("model", SVC(random_state=1))]),
+    SVC(kernel="linear", random_state=1),
+    Pipeline([("norm", Normalizer()), ("model", SVC(kernel="linear", random_state=1))]),
+    LinearRegression(),
+    LogisticRegression(),
```
Comment (giulio-tirabassi) [inline tests/test_estimator.py:18]: Maybe adding couple more regressor cases? so far it's just one.

**Review by giulio-tirabassi — COMMENTED** [review]
Just a minor comment. Also, did you check if we can more or less reproduce `Regression analysis with repeated samples` section from the tutorial? At least qualitatively.

File: tests/test_estimator.py (hunk 3 above)
Comment (carloszanella) [reply tests/test_estimator.py:18]: sure

**Pull request merged**